lib/sugar.rb

```diff
@@ -1,8 +1,10 @@
 require 'ronad/maybe'
 require 'ronad/just'
 require 'ronad/default'
+require 'ronad/eventually'
 
 def Maybe(value) ; Ronad::Maybe.new(value) end
 def Just(value); Ronad::Just.new(value) end
 def Default(fallback, value) ; Ronad::Default.new(fallback, value) end
+def Eventually(&b); Ronad::Eventually.new(b) end
```

Comment:
Eventually has the parameter name 'b'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)